src/pytest_workflow/plugin.py

```diff
@@ -133,6 +133,15 @@ def pytest_configure(config: PytestConfig):
     workflow_temp_dir = (
         Path(basetemp) if basetemp is not None
         else Path(tempfile.mkdtemp(prefix="pytest_workflow_")))
+
+    rootdir = Path(str(config.rootdir))
+    # Raise an error if the workflow temporary directory of the rootdir
+    # (pytest's CWD). This will lead to infinite looping and copying.
+    if str(workflow_temp_dir.absolute()).startswith(str(rootdir.absolute())):
```

Comment:
The condition could be shortened to `workflow_temp_dir.absolute().parent == rootdir.absolute()`

Reply:
But what if it is not its parent. But its parent parent?

Reply:
Valid point. Pathlib's [relative_to](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to) actually performs this check. Annoyingly, in our usecase, it raises an exception in the case the paths arent relative to one another. There seems to be no `is_relative_to` function.